integration_test/third_party_apps_test/applications/mssql/metadata.yaml

```diff
@@ -159,5 +148,5 @@ configuration_options:
           default: 60s
           description: A [time duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`.
         - name: receiver_version
-          default: "1"
+          default: null
```

Comment:
Why change this to `null`? I get that it might be more technically correct from a config schema perspective, but seeing as how these directly feed the docs I would place more importance on emphasizing to the user that leaving this field empty implies version 1.

Reply:
Good catch. I meant to change it back to "1" across the board and forgot. Done now.